lib/cc/yaml/nodes/engine_list.rb

```diff
@@ -2,7 +2,30 @@ module CC
   module Yaml
     module Nodes
       class EngineList < OpenMapping
+        GENERIC_ERROR_MESSAGE = "engines should be a map of engine name to engine config"
```

Comment:
I actually think we should move away from capitalizing `Engines` in these error messages: it isn't capitalized in some other ones that are autogenerated (by just using the key name). And since these are error messages about file contents, I think it's better to use the the actual key name in the file that is triggering the error. WDYT?

If that seems like sound reasoning, I can also change the existing cases to be consistent.